packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [28.1.0]
+
+### Added
+
+- Added token detection on 7 more networks ([#4184](https://github.com/MetaMask/core/pull/4184))
+
+### Changed
+
+- Bump `@metamask/keyring-controller` to ^16.0.0 ([#4234](https://github.com/MetaMask/core/pull/4171))
+- Bump `@metamask/keyring-api` to 6.0.0, `@metamask/eth-snap-keyring` to 4.0.0 and snap dependencies ([#4193](https://github.com/MetaMask/core/pull/4193))
+
+### Fixed
+
+- Fixed NFT detection after closing browser ([#4178](https://github.com/MetaMask/core/pull/4178))
```

Comment:
It's unclear to me what is being fixed here as well. I can see that it's related to NFT detection after the browser is closed, but the PR seems to just add initial constructor parameters. It's unclear to me how the additional parameters would be considered a bug fix

Reply:
Hmm, by looking at the PR description we should probably split this in multiple entries

Reply:
Yes I think so, good point